mlflow/projects/backend/loader.py

```diff
@@ -0,0 +1,32 @@
+import entrypoints
+import logging
+
+ENTRYPOINT_GROUP_NAME = "mlflow.mlproject_backend"
+
+__logger__ = logging.getLogger(__name__)
+
+
+# Statically register backend defined in mlflow
+MLFLOW_BACKENDS = dict()
+
+
+def load_backend(backend_name):
+    # Static backends
+    if backend_name in MLFLOW_BACKENDS:
+        return MLFLOW_BACKENDS[backend_name]()
```

Comment:
@jdlesage @smurching It seems reasonable for some backends to be referenced by URI rather than a human-readable name. For example, this would likely be the case if we introduced support for REST-based project backends. We may want to consider support for regex here was well (e.g., registering a backend to use for URIs of the form `http://.*` or `https://.*`.

Matching on the full backend name seems fine for now. If we were to add suppot of RESTful backends, it might be possible that

Reply:
I think your comment might have been cut off :P - I guess with the current scheme, we'd handle HTTP backends via a custom backend type, e.g. `mlflow-http` (probably named something better) and then the actual URI of the backend would be a field in the backend config

Reply:
Whoops! Accidentally left some trailing content at the end there. Ah, good point - putting the URL in the config sounds fine to me!